assets/cubyz/biomes/plateau/_defaults.zig.zon

```diff
@@ -0,0 +1,43 @@
+.{
+	.interpolation = .none,
+	.properties = .{},
+	.minHeightLimit = 7,
+	.minHeight = 50,
+	.maxHeight = 70,
```

Comment:
I think they stick out too much. Also you can use the `keepOriginalTerrain` trick to make them more consistently sized.
e.g. the following would set the height to 5-10 blocks above the remaining terrain:
```zig.zon
	.minHeight = 500,
	.maxHeight = 1000,
	.keepOriginalTerrain = 0.99,
```